pkg/api/admin.go

```diff
@@ -80,10 +82,12 @@ func (c *client) setupAdminRoutes() {
 			}
 		}
 
-		c.htmlResponseWrapper(ctx, http.StatusOK, "admin/index", gin.H{
+		ctx.HTML(http.StatusOK, "admin/index", gin.H{
 			"errors":     flashes,
 			"teams":      teamApps,
 			"gcpProject": c.gcpProject,
+			"loggedIn":   ctx.GetBool(middlewares.LoggedInKey),
+			"admin":      ctx.GetBool(middlewares.AdminKey),
```

Comment:
Kunne ikke det vært løst av enda en middleware? For nå må vi huske å sette denne alle steder.